### PR TITLE
Recommend

### PR DESCRIPTION
**Pull request by reactivepixel**

### CODE REVIEW

File: src/models/recommend.js [hunk 1]
```diff
@@ -7,36 +7,135 @@ module.exports = function(express) {
...
+      },
+    ])
+  })
+  .then( function(express) {
+    // What things might alice like?
+    // return ger.recommendations_for_person('movies', express, {actions: {likes: 1}})
+    return console.log(express);
+
+  })
+  // .then( function(recommendations) {
```
Comment (reactivepixel) [inline src/models/recommend.js]: This is a lot of commented out code. Please remove completely .

File: src/models/recommend.js [hunk 2]
```diff
@@ -1,42 +1,48 @@
 module.exports = function(express) {
-
-//   var express=require('express');
-//   var router = express.Router();
-//   var history =require('./history');
-//   var db =require('./db');
   var g = require('ger');
```
Comment (reactivepixel) [inline src/models/recommend.js]: g is ambiguous. Please change to be semantic.

File: src/models/recommend.js [hunk 3]
```diff
@@ -1,42 +1,48 @@
...
   var esm = new g.MemESM();//Event Store Manager
```
Comment (reactivepixel) [inline src/models/recommend.js]: Comment formatting is bad.

File: src/routes/api/recommend.js [hunk 4]
```diff
@@ -5,7 +5,11 @@
...
 
+var g = require('ger');
+var esm = new g.MemESM();//Event Store Manager
+var ger = new g.GER(esm);//Good Enough Recommender----Added to package.json
 var buckets=require('../../models/recommend');
 
+var rec=require('../../models/rec');
 
-// console.log(buckets());
+buckets();
```
Comment (reactivepixel) [inline src/routes/api/recommend.js]: Logic doesn't make sense.

File: src/routes/index.js [hunk 5]
```diff
@@ -40,6 +40,13 @@ module.exports = function(express) {
   // router.use('/api/', require('./api/recommend')(express));
 
   // MIDDLE-WARE
+
+
+  // router.use('/api/', require('./api/recommend')(express));
```
Comment (reactivepixel) [inline src/routes/index.js]: Remove Commented Code

File: src/server.js [hunk 6]
```diff
@@ -21,6 +21,8 @@ if (!process.env.PORT) dotenv = require('dotenv').load(); // If
 
 
 
+
+
```
Comment (reactivepixel) [inline src/server.js]: Remove

File: src/models/db.js [hunk 7]
```diff
@@ -287,7 +278,13 @@ module.exports = function() {
     foreignKey: 'user_id'
   });
 
-  //belongsToMany - creates an N:M association with a join table and adds plural association mixins to the source. The junction table is created with sourceId and targetId.
+  /**
+   *
+   * @var {attributes} .belongsToMany
```
Comment (jc-roll) [inline src/models/db.js:286]: The variable is not called .belongsToMany it is belongsToMany the "." just shows the separation #48

File: src/models/recommend.js [hunk 8]
```diff
@@ -1,7 +1,17 @@
...
+
+  /**
+   * @var {connection} esm
+   *   Event Store Manager
+   */
+  var esm = new g.MemESM();
+
+  /**
+   * @var {connection} ger
+   *   Good Enough Recommender----Added to package.json
```
Comment (jc-roll) [inline src/models/recommend.js]: The second half of this description could confuse people I think you should explain what the Good Enough Recommend-er actually does. #48

File: src/models/db.js [hunk 9]
```diff
@@ -30,6 +30,8 @@ module.exports = function() {
   // =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 
   /**
+   *
+   * @var {connection} .define
```
Comment (jc-roll) [inline src/models/db.js]: var is define not .define #48

**Pull request merged**